finality-aleph/src/party/mod.rs

```diff
@@ -39,12 +39,15 @@ use sp_runtime::traits::{Block, Header};
 use std::{collections::HashSet, default::Default, marker::PhantomData, sync::Arc, time::Duration};
 
 mod task;
+
 use task::{Handle, Task};
+
```

Comment:
In other modules we put all these `use`s below a block of `mod`s, the sudden empty lines here are kinda ugly, so I would prefer if you did that.

The empty line below, separating the `mod`s from the `use`s is a fine addition ofc.

Reply:
I think fixed, could you check if this is what you meant?